### PR TITLE
144 update readme

### PR DESCRIPTION
**Pull request by garrett-vangilder**
## Status
**READY**

## Documentation

[X] I have fully documented features in this PR's code. Please put an `X` in the box to confirm.

[ ] This PR does **NOT** require documentation.

## Description
A few sentences describing the overall goals of the pull request's commits.

This updates the README.md per Ryan's comments.

## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.

```sh

```

## Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.

1. Look at the README.md This code does not execute.

## Impacted Files in Application
List general components of the application that this PR will affect:

*

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,27 @@
...
+If you need to download dotnet on to your local machine, please visit [Microsoft's Documentation](https://www.microsoft.com/en-us/download/details.aspx?id=30653)
+
+2. bower
+
+If you need to download bower on to your local machine, please visit [Bower's Documentation](https://bower.io/)
+
 ## Installation
 
 Clone or fork the project. Navigate to where the project is saved on your machine.
+Please note that if running the application in OSX/UNIX/Windows that the commands below require for you to manually change the path to your database.
```
Comment (matthamil) [inline README.md]: "Please note that if running the application in OSX/UNIX/Windows "

--> "Please note that you must manually change the path to your database before running the commands below"

File: README.md [hunk 2]
```diff
@@ -1,12 +1,27 @@
...
+In other words, if you copy and paste the following commands they will not initially work until you specifically locate your database file and place the path to said file into the following commands.
```
Comment (matthamil) [inline README.md]: Redundant line

File: README.md [hunk 3]
```diff
@@ -1,12 +1,27 @@
...
 ## Installation
 
 Clone or fork the project. Navigate to where the project is saved on your machine.
+Please note that if running the application in OSX/UNIX/Windows that the commands below require for you to manually change the path to your database.
+In other words, if you copy and paste the following commands they will not initially work until you specifically locate your database file and place the path to said file into the following commands. 
+
+### OSX/UNIX
 
-Run the following commands to get the project running:
+Please run the following commands in your terminal.
```
Comment (matthamil) [inline README.md]: Omit "please," it's fluff/unnecessary

File: README.md [hunk 4]
```diff
@@ -16,7 +31,10 @@ bower install
 dotnet run
 ```
 
-Windows
+###Windows
+
+Please run the following commands in Windows Powershell
```
Comment (matthamil) [inline README.md]: Omit "please"

File: README.md [hunk 5]
```diff
@@ -1,12 +1,27 @@
 # Bangazon Web Initial Site
 
+## Dependencies
+
+To ensure that the Initial Bangazon App works as intended make sure that you have the following dependencies and technologies on your local machine
```
Comment (matthamil) [inline README.md:5]: "Initial Bangazon App relies on the following dependencies to work properly on your machine"

Not sure if it matters but you started a sentence with an infinitive ¯\_(ツ)_/¯

File: README.md [hunk 6]
```diff
@@ -1,12 +1,27 @@
...
+
+1. dotnet 
+
+If you need to download dotnet on to your local machine, please visit [Microsoft's Documentation](https://www.microsoft.com/en-us/download/details.aspx?id=30653)
```
Comment (matthamil) [inline README.md]: onto*

**Review by matthamil — CHANGES_REQUESTED** [review]
Minor fixes

File: README.md (hunk 3 above)
Comment (MattKraatz) [reply README.md]: :+1:

File: README.md (hunk 2 above)
Comment (MattKraatz) [reply README.md]: :+1:

**Comment by garrett-vangilder** [comment]
@matthamil made changes.

**Review by MattKraatz — APPROVED** [review]
LGTM

**Review by matthamil — APPROVED** [review]

**Comment by daynewright** [comment]
LGTM

**Comment by Anulfo** [comment]
LGTM

**Pull request merged**